### PR TITLE
Added tests for lscfg and lsvpd tools

### PR DESCRIPTION
**Pull request by charishm**
lscfg command displays  configuration, diagnostic, and vital product data
(VPD) information about the system.
    
lsvpd utility lists device Vital Product Data (VPD), which includes the
following information and more: vendor, version, revision level, serial number.

Signed-off-by: charishma <charishm@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@charishm please squash commit message in a single commit as well rebase as make main script pylint compliant

File: generic/ras.py [hunk 1]
```diff
@@ -274,6 +275,65 @@ def test12_rtas_errd_and_rtas_dump(self):
...
+            self.run_cmd("lsvpd --path=%s" % copyfile_path)
+
+        path_zip = process.system_output("find /var/lib/lsvpd/ -iname vpd.*.gz"
+                                         " | head -1", shell=True).strip()
+        if path_zip:
+            self.run_cmd("lsvpd --zip=%s" % path_zip)
+
+        if self.is_fail >= 1:
+            self.fail("%s command(s) failed in lscfg tool verification"
+                      % self.is_fail)
```
Comment (narasimhan-v) [inline generic/ras.py]: Since this part of code is repeated, have a helper function to do it.

**Comment by PraveenPenguin** [comment]
@charishm any update here

File: generic/ras.py [hunk 2]
```diff
@@ -274,6 +275,65 @@ def test12_rtas_errd_and_rtas_dump(self):
...
+        cpu_output = process.system_output('cat /proc/cpuinfo')
+        if "emulated by qemu" in cpu_output:
+            raise exceptions.TestWarn(
+                "Not supported on the pSeries KVM Guest platform")
+
+        self.run_cmd("lscfg -h")
+        self.run_cmd("lscfg -V")
+        self.run_cmd("lscfg -v")
+        self.run_cmd("lscfg -D")
+        self.run_cmd("lscfg -p")
```
Comment (balamuruhans) [inline generic/ras.py]:
```
for option in ['-h', '-V', '-v', '-D', '-p']:
    self.run_cmd("lscfg %s" % option)
```

File: generic/ras.py [hunk 3]
```diff
@@ -274,6 +275,65 @@ def test12_rtas_errd_and_rtas_dump(self):
...
+        cpu_output = process.system_output('cat /proc/cpuinfo')
+        if "emulated by qemu" in cpu_output:
+            raise exceptions.TestWarn(
+                "Not supported on the pSeries KVM Guest platform")
+
+        self.run_cmd("lsvpd -h")
+        self.run_cmd("lsvpd -D")
+        self.run_cmd("lsvpd -v")
+        self.run_cmd("lsvpd --serial")
+        self.run_cmd("lsvpd --type")
```
Comment (balamuruhans) [inline generic/ras.py]: same here

File: generic/ras.py [hunk 4]
```diff
@@ -20,6 +20,7 @@
 from avocado import main
 from avocado.utils import process
 from avocado.utils.software_manager import SoftwareManager
+from avocado.core import exceptions
```
Comment (apahim) [inline generic/ras.py]: Please don't use internal APIs. They have no guarantee of stability and are intended for internal usage (well, or hacking), not for test writing. In this case, as you're using it for `TestWarn` exception raising, you can use `self.log.warn()` instead.

**Comment by PraveenPenguin** [comment]
@charishm any update here

**Comment by PraveenPenguin** [comment]
@charishm  it is pending from more than 6 month as no update if you still want to pursue this please raise new PR , closing this PR

**Pull request closed without merging**